### PR TITLE
add some tests for codeigniter config

### PR DESCRIPTION
**Pull request by esetnik**

### CODE REVIEW

File: application/controllers/Set_config.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+class Set_config extends CI_Controller
+{
+	public function config_item()
+	{
+    $config_key = $this->input->get('key');
```
Comment (kenjis) [inline application/controllers/Set_config.php:7]: Please fix indentation. I use tabs.

File: application/tests/controllers/Set_config_test.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+					}
+				);
+
+				$output = $this->request(
+					'GET',
+					['Set_config', 'config_item'],
+					['key' => $config_key]
+				);
+				$this->assertEquals($output, $config_value);
+		}
```
Comment (kenjis) [inline application/tests/controllers/Set_config_test.php]: Please fix indentation. You put two tabs, but only one is needed.

File: application/tests/controllers/Set_config_test.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+{
+		public function test_set_config()
+		{
+				$config_key = 'test_key';
+				$config_value = 'test_value';
+				$this->request->setCallable(
+					function ($CI) use ($config_key, $config_value) {
+						$CI->config->set_item($config_key, $config_value);
+					}
+			  );
```
Comment (kenjis) [inline application/tests/controllers/Set_config_test.php]: Please remove spaces.

**Comment by esetnik** [comment]
Sure I will fix indentation

**Comment by kenjis** [comment]
I added fixes: https://github.com/kenjis/ci-app-for-ci-phpunit-test/commit/502458a939eecc7c7613db56f24de65af08130c1

**Comment by esetnik** [comment]
I wasn't sure if you would want the `.editorconfig` as it's a common way to ensure that most people's editors will automatically use the code style that you like for the PRs they submit. Thanks for merging.

**Comment by kenjis** [comment]
I removed `.editorconfig`, but I will add it again.
I did not know `.editorconfig`. Thank you for your information.

**Pull request merged**